src/TSTransformer/nodes/binding/transformObjectBindingPattern.ts

```diff
@@ -33,7 +43,10 @@ export function transformObjectBindingPattern(
 			// if name is not identifier, it must be a binding pattern
 			// in that case, prop is guaranteed to exist
 			assert(prop);
-			const value = objectAccessor(state, parentId, state.getType(bindingPattern), prop);
+			const value = isSpread
+				? spreadDestructObject(state, parentId, preSpreadNames)
```

Comment:
`spreadDestructObject(state, parentId, preSpreadNames)` does not appear to be covered by unit tests